charts/substra-backend/templates/statefulset-worker.yaml

```diff
@@ -35,6 +35,8 @@ spec:
         app.kubernetes.io/component: substra-worker
         role-api-k8s-client: 'true'
         role-database-client: 'true'
+        # Needed for downloading models from other backends + connecting to remote registry
+        role-internet-egress: 'true'
```

Comment:
Do we have leads on why this was working both with skaffold and on substra-dev but not on staging?

Reply:
Was the post-mortem not clear enough?

Reply:
Very clear, thank you. Missed the thought process of it somehow when I first read it.